sktime/regression/tests/test_all_regressors.py

```diff
@@ -78,5 +78,8 @@ def test_regressor_output(self, estimator_instance, scenario):
 
         # check predict
         assert isinstance(y_pred, np.ndarray)
-        assert y_pred.shape == (X_new_instances,)
+        assert y_pred.shape == (X_new_instances,) or y_pred.shape == (
+            X_new_instances,
+            y_pred.ndim,
```

Comment:
same as above

Reply:
`y_pred.ndim` should be `y_train.shape[1]` iff `y_train.ndim > 1`.